src/logic/wallets/transactionDataCheck.ts

```diff
@@ -1,18 +1,19 @@
+import { loadFromStorage, saveToStorage } from 'src/utils/storage'
+
+const LEDGER_TRANSACTION_DATA = 'TRANSACTION_DATA'
+const WALLET = 'Ledger'
```

Comment:
can be replaced with `WALLET_PROVIDER.LEDGER` from `src/logic/wallets/getWeb3.ts`